glide.yaml

```diff
@@ -64,3 +64,5 @@ import:
   version: fa7b53cdfc9105c70f134574002f406232921437
 - package: github.com/spf13/afero
   version: v1.2.2
+- package: github.com/cdrage/uilive
```

Comment:
why forked version?

Reply:
Needed to implement a few open PR's that the maintainer has yet to merge in: https://github.com/gosuri/uilive/pull/29 so using a forked version with those changes.